charts/umbrella/Chart.yaml

```diff
@@ -50,12 +50,12 @@ dependencies:
   - condition: discoveryfinder.enabled
     name: discoveryfinder
     repository: https://eclipse-tractusx.github.io/charts/dev
-    version: 0.2.2
+    version: 0.2.5
     # bpn-discovery
   - condition: bpndiscovery.enabled
     name: bpndiscovery
     repository: https://eclipse-tractusx.github.io/charts/dev
-    version: 0.2.2
+    version: 0.2.6
```

Comment:
would be great if you could use the occasion to finalize the [bpndiscovery configuration](https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-0.16.0/charts/umbrella/values.yaml#L462) with ingress, idp and so on, I think for the [discoveryservice](https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-0.16.0/charts/umbrella/values.yaml#L487) we already did that

Reply:
Done. 
Added ingress and idp.